code/modules/mod/modules/modules_maint.dm

```diff
@@ -182,6 +182,49 @@
 	mod.wearer.put_in_hands(balloon)
 	drain_power(use_power_cost)
 
+/// Paper dispenser - dispenses (sometimes burning) paper sheets
```

Comment:
nitpick but the other modules have both words capitalized and a dot at the end in their comment